src/Parsers/IAST.h

```diff
@@ -195,12 +195,12 @@ class IAST : public std::enable_shared_from_this<IAST>, public TypePromotion<IAS
         bool one_line;
         bool always_quote_identifiers = false;
         IdentifierQuotingStyle identifier_quoting_style = IdentifierQuotingStyle::Backticks;
-        bool show_secrets = true; /// Show secret parts of the AST (e.g. passwords, encryption keys).
+        bool show_secrets = false; /// Show secret parts of the AST (e.g. passwords, encryption keys).
```

Comment:
Since you're initializing show_secrets in the FormatSettings constructor, there's no need to initialize it here, just like with the `one_line`.

It's better to only initialize something once, for consistency. Say, if someone wanted to change that default value, someone will have to do that twice, or someone could even forget to change one of the places and keep the reader slightly perplexed. Moreover, this is exactly what happened here, and you personally 'suffered' the consequences of having to edit the same default value twice :)